test/repositories/dbPing.spec.js

```diff
@@ -0,0 +1,64 @@
+const { expect } = require("chai");
+const Chance = require("chance");
+const sinon = require("sinon");
+const { getPings } = require("../../repositories/pings");
+const pool = require("../../repositories/pool");
```

Comment:
sorry @austinjw we refactored the calls from repositories/pool into repositories/databaseOps. you will need to update this here